.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
   - redis-server
 language: go
 go:
-  - 1.13.1
+  - 1.14
```

Comment:
May we use 1.14.0 as a docker image version and travis version?

Reply:
Sounds reasonable.

Reply:
Doesn't work for Travis, though.

Reply:
It's strange as I thought that travis is using docker images with their versioning as a build environment. By the way let's merge this PR and will fix travis version later when version with patch number will appear.